Extension/src/LanguageServer/ui.ts

```diff
@@ -238,15 +239,15 @@ export class LanguageStatusUI {
         if (this.isParsingWorkspacePaused) {
             const displayTwoStatus: boolean = this.isParsingFiles && this.isParsingWorkspace;
             return (this.isParsingFiles ? this.parsingFilesTooltip : "")
-                + (displayTwoStatus ? " | " : "")
```

Comment:
This indentation seems incorrect? It's to the left of the start of the expression on the previous line.

Reply:
Weird. All three of these got their indentation broken. The linter/formatter isn't triggering on them, and I have no idea why git would have messed with the indentation. 

Hmm. Unless it happened when the formatter did kick in when I hit save and there was a merge conflict (which puts stuff like  `>>>>>>` in the text).

Reply:
fixed.